### PR TITLE
feat: add the February and March review blog

### PR DESCRIPTION
**Pull request by kuomc**
This PR adds the review blog of the sciwork events during February and March in 2024.

### CODE REVIEW

**Comment by kuomc** [comment]
Hi @yungyuc , I would appreciate your feedback whenever you have a moment.

File: content/blog/2024/202402-03-review.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+developed many plotting related features to make the presentation of the mesh more flexible.
+
+In addition to project development, there were also many people having questions about their work.
+Some people asked about the difficulties they encountered when working on the Vue.js project and
+auto-streaming databases, and some people asked how to improve the performance of LLM models from
+both software and hardware perspectives.
+
+March Workshop
+--------------
+Sciwork organized a new event called Advanced CFD software development workshop in March. Chun-Hsu and
```
Comment (yungyuc) [inline content/blog/2024/202402-03-review.rst:22]: Please keep sciwork lower-case everywhere.

File: content/blog/2024/202402-03-review.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+many sciwork volunteers have been preparing for this workshop since February. Yung-Yu introduced the
+numerical method space-time CESE method, unstructured mesh with mixed elements, and many important
+software engineering techniques to make participants understand how to write the programs to get correct
+results with high-performance and maintainable numerical software through the modmesh project.
+
+Mesh has always been an important issue in numerical software that is easier ignored. While preparing for
```
Comment (yungyuc) [inline content/blog/2024/202402-03-review.rst:28]: *is easier ignored* does not sound right.  Maybe you meant *easily ignored*?

File: content/blog/2024/202402-03-review.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+this workshop, modmesh contributors make a lot of effort to finish plotting the unstructured mesh with
+mixed elements. The participants can see a more concrete mesh, which is pretty important for the development
```
Comment (yungyuc) [inline content/blog/2024/202402-03-review.rst:30]: *development of numerical software development* does not sound right.

File: content/blog/2024/202402-03-review.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+of numerical software development.
+
+After this workshop many participants discussed on the discord channels to continue to learn about the numerical
+software modmesh. Some participants joined the March scisprint to learn more about the numerical software. sciwork
```
Comment (yungyuc) [inline content/blog/2024/202402-03-review.rst:34]: It may be more concise to write just *modmesh*.

File: content/blog/2024/202402-03-review.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+will continue to organize more events to enable more people who are interested in numerical software to write
+programs and share ideas together.
+
+February and March Scisprint
+----------------------------
+Two scisprint events were held in February and March, one in Hsinchu in February and one in Taipei in March.
+After the CFD workshop in March, the scisprint in March welcomed many new partners. Everyone started to discuss
+and develop projects of their own interest, and some people came to communicate with each other about their lives and works.
+
+After the CFD workshop in March, modmesh welcomed many new friends. While building modmesh with the new friends, modmesh
```
Comment (yungyuc) [inline content/blog/2024/202402-03-review.rst:44]: The sentence of *After the ...* seems to duplicate with the previous paragraph.

**Review by yungyuc — COMMENTED** [review]
Thanks, @kuomc .  I added some comments for the file.

File: content/blog/2024/202402-03-review.rst (hunk 1 above)
Comment (kuomc) [reply content/blog/2024/202402-03-review.rst]: Miss one. I fixed it.

File: content/blog/2024/202402-03-review.rst (hunk 3 above)
Comment (kuomc) [reply content/blog/2024/202402-03-review.rst:30]: I fixed it.

File: content/blog/2024/202402-03-review.rst (hunk 4 above)
Comment (kuomc) [reply content/blog/2024/202402-03-review.rst]: I agree.

File: content/blog/2024/202402-03-review.rst (hunk 5 above)
Comment (kuomc) [reply content/blog/2024/202402-03-review.rst]: I changed the sentence pattern.

**Comment by kuomc** [comment]
> Thanks, @kuomc . I added some comments for the file.

I fixed them. Please help me confirm the results.

**Comment by yungyuc** [comment]
LGTM.  I'll let you merge it yourself in case you want to fix minor wording issues.

**Review by yungyuc — APPROVED** [review]
LGTM. I'll let you merge it yourself in case you want to fix minor wording issues.

**Pull request merged**